blogposts/2020/announcing-sourcegraph-3.17.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Sourcegraph 3.17: Faster and automatic precise code intelligence, preview of code insights, and AND/OR queries for searching file contents"
+author: Adam Herzog
+publishDate: 2020-06-20T10:00-07:00
+tags: [
+  blog
+]
+slug: sourcegraph-3.17
+heroImage: /blog/3.17-release-blog-img.jpg
+published: true
+---
+
+Explore, navigate, and better understand all code, everywhere, faster with Sourcegraph Universal Code Search. Uber, Yelp, Nutanix, Tinder, and other enterprises rely on Sourcegraph to improve developer productivity and manage large-scale code changes.
+
+<div style="padding-left: 2rem">
+
+[**🧠 Faster and automatic precise code intelligence**](#faster-and-automatic-precise-code-intelligence)<br />
+Precise code intelligence is now 35% faster than in 3.16. Automatic precise code intelligence (🧪 experimental feature) will bring precise results to more repositories faster.
+
+[**📊 Product preview: Code insights**](#product-preview-code-insights)<br />
+Help developers be more productive and do their jobs better by aggregating data in Sourcegraph.
```

Comment:
I am worried this sentence will be interpreted incorrectly that we will be tracking data about developers. I want to focus the data aggregation on the code. 
```suggestion
Understand high-level patterns of your code, and track code changes over time by aggregating data in Sourcegraph.
```

Reply:
great suggestion - totally agree